openjdk.build/playlist.xml

```diff
@@ -4,7 +4,8 @@
 		<variations>
 			<variation></variation>
 		</variations>
-		<command>cd $(OPENJDK_SYSTEMTEST_ROOT)/openjdk.build && make test.ClassloadingLoadTest  JAVA_ARGS="$(JVM_OPTIONS)"</command>
+		<command>cd $(OPENJDK_SYSTEMTEST_ROOT)/openjdk.build && make test.ClassloadingLoadTest JAVA_ARGS=$(Q)$(JVM_OPTIONS)$(Q); \
```

Comment:
; \ is not valid in a Windows shell.  Instead you could create a macro AND_THEN (or a similar name) with a value of '&' on windows, otherwise ';'.  and then use cd $(OPENJDK_SYSTEMTEST_ROOT)/openjdk.build && make test.ClassloadingLoadTest JAVA_ARGS=$(Q)$(JVM_OPTIONS)$(Q) $(AND_THEN) $(TEST_STATUS)

Reply:
See https://github.com/eclipse/openj9/issues/490 for the required missing macro.  We can go ahead with the merge for now because we are not yet testing Windows.
See https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/25 for the follow on issue to change the playlist to use the macro when it exists.
--



--